yang/ietf-l2vpn-ntw.yang

```diff
@@ -140,16 +140,91 @@ module ietf-l2vpn-ntw {
   /* Typedefs */
     /* 
 
+  typedef esi-type {
```

Comment:
Thanks, Samier. 

Please transform the enumeration into identities. This will save us some cycles with yangdoctors review.

Reply:
Sure! Please see 5bb762cb4ea07eb3cbd4e92efbbb89514c23474a